util/config/local.go

```diff
@@ -0,0 +1,55 @@
+package config
+
+import (
+	"os/user"
+	"path"
+
+	"github.com/argoproj/argo-cd/util/cli"
+)
+
+// LocalConfig holds all local session config.
+type LocalConfig struct {
+	Sessions map[string]string `json:"sessions"`
+}
+
+// LocalConfigDir returns the local configuration path for settings such as cached authentication tokens.
+func localConfigDir() (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+	return path.Join(usr.HomeDir, ".argocd"), nil
+}
+
+// LocalConfigPath returns the local configuration path for settings such as cached authentication tokens.
+func localConfigPath() (string, error) {
+	dir, err := localConfigDir()
+	if err != nil {
+		return "", err
+	}
+	return path.Join(dir, "config"), nil
+}
+
+// ReadLocalConfig loads up the local configuration file.
+func ReadLocalConfig() (LocalConfig, error) {
+	var config LocalConfig = LocalConfig{
```

Comment:
unused variable

Reply:
I'm not sure I understand which variable is unused, since we're returning the config here.